vision/automl/edge_container_predict/README.md

```diff
@@ -0,0 +1,50 @@
+# AutoML Vision Edge Container Prediction
+
+This is an example to show how to predict with AutoML Vision Edge Containers.
+The test (freeze_meta_graph_lib_test.py) shows an automatical way to run the
+prediction.
+
+If you want to try the test manually with a sample model, please install
+[gsutil tools](https://cloud.google.com/storage/docs/gsutil_install) and
+[Docker CE](https://docs.docker.com/install/) first, and then follow the steps
+below. All the following instructions with commands assume you are in this
+folder.
+
++   Step 1. Pull the Docker image.
+
+```bash
+$ CPU_DOCKER_GCS_PATH=gcr.io/automl-vision-ondevice/gcloud-container-1.12.0:latest
+$ sudo docker pull ${CPU_DOCKER_GCS_PATH}
+```
+
++   Step 2. Get a sample saved model.
+
+```bash
+$ SAMPLE_SAVED_MODEL=gs://cloud-samples-data/vision/edge_container_predict/saved_model.pb
+$ mkdir model_path
+$ YOUR_MODEL_PATH=$(realpath model_path)
+$ gsutil -m cp ${SAMPLE_SAVED_MODEL} ${YOUR_MODEL_PATH}
+```
+
++   Step 3. Run the Docker container.
+
+```bash
+$ CONTAINER_NAME=AutomlVisionEdgeContainerPredict
+$ PORT=8501
+$ sudo docker run --rm --name ${CONTAINER_NAME} -p ${PORT}:8501 -v ${YOUR_MODEL_PATH}:/tmp/mounted_model/0001 -t ${CPU_DOCKER_GCS_PATH}
```

Comment:
can we use two different values for the ports here?  especially since in the next step the port number should match with one of these, it might help the developer remember which port is which in port forwarding.

Reply:
The port should be exactly the same. I updated the doc a little bit to clarify this.

Reply:
I mean the part of `-p ${PORT}:8501`, where `${PORT}` on local host gets mapped to port `8501` of the running container.  These two do not need to be the same if I understand this correctly.

Reply:
Got it. Yes, you are right. I changed it to a different number.